docs/SPEC.md

```diff
@@ -1235,8 +1235,8 @@ The generator is then used to produce the set of intervals.
 The set of intervals will include all intervals that intersect with the initial range of time.
 The `intervals` function is designed to be used with the `intervals` parameter of the `window` function.
 
-By default the end boundary of an interval will align with the Unix epoch (zero time) modified by the offset of the `location` option.
```

Comment:
I don't think we've ever implemented a location parameter.

Reply:
We will implement it sometime. Its a TODO for now.